api.planx.uk/modules/pay/service/inviteToPay/sendConfirmationEmail.test.ts

```diff
@@ -60,12 +60,12 @@ describe("sendAgentAndPayeeConfirmationEmail", () => {
     });
 
     const expectedConfig = {
+      emailReplyToId: "123",
```

Comment:
🐛 This PR highlighted this issue in a few places.

`emailReplyToId` should be part of the config, not a part of the personalisation ([docs](https://docs.notifications.service.gov.uk/node.html#send-an-email-method)). In some tests where we're actually calling the test instance of the notify client it also needs to be both a valid UUID, as well as a valid `emailReplyToId` from our account.

This explains why we've previously had issue with applicants replying to an incorrect email address.